content/en/docs/languages/java/api-components.md

```diff
@@ -0,0 +1,1690 @@
+    ContextPropagators propagators = openTelemetry.getPropagators();
+  }
+}
+```
+<!-- prettier-ignore-end -->
+
+### GlobalOpenTelemetry
+
+[GlobalOpenTelemetry](https://www.javadoc.io/doc/io.opentelemetry/opentelemetry-api/latest/io/opentelemetry/api/GlobalOpenTelemetry.html)
+holds a global singleton [OpenTelemetry](#opentelemetry) instance.
+
+Instrumentation should avoid using `GlobalOpenTelemetry`. Instead, accept
+`OpenTelemetry` as an initialization argument and default to the
+[Noop implementation](#noop-implementation) if not set. There is an exception to
+this rule: the `OpenTelemetry` instance installed by the
+[Java agent](/docs/zero-code/java/agent/) is available via
+`GlobalOpenTelemetry`. Users with additional manual instrumentation are
+encouraged to access it via `GlobalOpenTelemetry.get()`.
+
+`GlobalOpenTelemetry.get()` is guaranteed to always return the same result. If
+`GlobalOpenTelemetry.get()` is called before `GlobalOpenTelemetry.set(..)`,
+`GlobalOpenTelemetry` is set to the noop implementation and future calls to
+`GlobalOpenTelemetry.set(..)` throw an exception. Therefore, it's critical to
+call `GlobalOpenTelemetry.set(..)` as early in the application lifecycle as
+possible, and before `GlobalOpenTelemetry.get()` is called by any
+instrumentation. This guarantee surfaces initialization ordering issues: calling
+`GlobalOpenTelemetry.set()` too late (i.e. after instrumentation has called
+`GlobalOpenTelemetry.get()`) triggers an exception rather than silently failing.
+
+If [autoconfigure](../configuration/#zero-code-sdk-autoconfigure) is present,
+`GlobalOpenTelemetry` can be automatically initialized by setting
+`-Dotel.java.global-autoconfigure.enabled=true` (or via env var
+`export OTEL_JAVA_GLOBAL_AUTOCONFIGURE_ENABLED=true`). When enabled, the first
+call to `GlobalOpenTelemetry.get()` triggers autoconfiguration and calls
+`GlobalOpenTelemetry.set(..)` with the resulting `OpenTelemetry` instance.
+
+The following code snippet explores `GlobalOpenTelemetry` API context
+propagation:
+
+<!-- prettier-ignore-start -->
+<?code-excerpt "src/main/java/otel/GlobalOpenTelemetryUsage.java"?>
+```java
+package otel;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.api.OpenTelemetry;
+
+public class GlobalOpenTelemetryUsage {
+
+  public static void openTelemetryUsage(OpenTelemetry openTelemetry) {
+    // Set the GlobalOpenTelemetry instance as early in the application lifecycle as possible
+    // Set must only be called once. Calling multiple times raises an exception.
+    GlobalOpenTelemetry.set(openTelemetry);
+
+    // Get the GlobalOpenTelemetry instance.
+    openTelemetry = GlobalOpenTelemetry.get();
+  }
+}
+```
+<!-- prettier-ignore-end -->
+
+### TracerProvider
+
+[TracerProvider](https://www.javadoc.io/doc/io.opentelemetry/opentelemetry-api/latest/io/opentelemetry/api/trace/TracerProvider.html)
+is the API entry point for traces, and provides [Tracers](#tracer). See
```

Comment:
```suggestion
is the API entry point for traces and provides [Tracers](#tracer). See
```

Reply:
Applying the same to metrics and logs sections. 👍